database/migrations/20221231090036-create-users-table.js

```diff
@@ -13,11 +13,11 @@ module.exports = {
       },
       name: {
         type: Sequelize.STRING,
-        allowNull: true,
+        allowNull: false,
       },
       email: {
         type: Sequelize.STRING,
-        allowNull: true,
+        allowNull: false,
       },
       status: {
         type: Sequelize.ENUM("Pending", "Approved", "Rejected"),
```

Comment:
keep these enums somewhere in constants. I think there is a way to directly insert array of enums in sequelize, don't remember exact thing.
But you can import Enums array herere & spread it,,,, `...typeArr`